packages/components/src/Illustration/Spot/_docs/Spot.mdx

```diff
@@ -0,0 +1,45 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as StickerSheetStories from "./Spot.stickersheet.stories"
+import * as SpotIllustrationStories from "./Spot.stories"
+
+<Meta of={SpotIllustrationStories} />
+
+# Spot Illustration
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Illustration"
+  figma="https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-UI-Kit%3A-Heart?type=design&node-id=50176%3A251125&mode=design&t=5GnjF9A8Wnd9UmHY-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082060340/Illustration"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { CompanyDetails } from "@kaizen/components"'
+/>
+
+## Overview
+
+Spot illustrations gives visual context to tasks. Most illustrations on our platform are presentational and should use the default empty alt text (ie: `alt=""`) provided by the base component.
+
+<Canvas of={SpotIllustrationStories.Playground} />
+<Controls of={SpotIllustrationStories.Playground} />
+
+## API
+
+### Enabled Aspect Ratio
+
+When enabled this wraps the image in a `figure` tag and set the aspect ratio to 1/1.
+
+<Canvas of={SpotIllustrationStories.EnabledAspectRatio} />
+
+When not enabled, the `img` tag will attempt to fill a width of 100%.
+
+## All Spot illustrations
```

Comment:
I've included the stickersheet that has a list of the illustrations since our consumers would not be able to see the hidden stories on prod.

Reply:
I think just to reiterate, put an example of the import statement before the canvas

Reply:
Good idea, I'll make a quick update to include that example